setup.cfg

```diff
@@ -51,6 +51,7 @@ install_requires =
     holoviews
     dash~=2.11.1
     chromadb~=0.4.22
+    weaviate-client~=4.6.1
```

Comment:
It looks there is some incompatibility with weaviate and protobuf, as indicated in the failing tests: https://github.com/Semantics-of-Sustainability/tempo-embeddings/actions/runs/9400532350/job/25890343632
I suppose this requires some detailed investigation and manual specification of dependency version.
Does it build locally when you use these dependencies?

Reply:
yes, I ran the tests locally and everything works. Perhaps is some incompatibility with the linux version? I have no clue honestly of how to fix it...

Reply:
I cannot check now, but could it be that you have some older package versions installed from previously?
To reproduce, perhaps try:
```console
python3 -m pip install --upgrade pip setuptools
python3 -m pip install --upgrade .[dev,publishing]
```
And see if you can still run the tests then.

Reply:
Looks like [this commit](https://github.com/Semantics-of-Sustainability/tempo-embeddings/pull/54/commits/af5ce5df4220d65eaa4f3a7c6e988b90d6c4345c) has solved it; see e.g. [this run](https://github.com/Semantics-of-Sustainability/tempo-embeddings/actions/runs/9449949196/job/26027430330?pr=54).